docs/hazi/3-felhasznaloi-felulet-kialakitasa/index.md

```diff
@@ -32,7 +32,15 @@ A képernyőképeket a megoldás részeként kell beadni, a repository-d gyöké
 A képernyőképek így felkerülnek GitHub-ra a git repository tartalmával együtt.
 Mivel a repository privát, azt az oktatókon kívül más nem látja.
 Amennyiben olyan tartalom kerül a képernyőképre, amit nem szeretnél feltölteni, kitakarhatod a képről.
-- :exclamation: A beadott megoldások mellé külön indoklást, illetve leírást nem várunk el, ugyanakkor az elfogadás feltétele, hogy a beadott kódban a feladat megoldása szempontjából relevánsabb részek kommentekkel legyenek ellátva.
+- :exclamation: Ehhez a feladathoz érdemi előellenőrző nem tartozik: minden push után lefut ugyan, de csak a neptun.txt kitöltöttségét ellenőrzi és azt, van-e fordítási hiba. Az érdemi ellenőrzést a határidő lejárta után a laborvezetők teszik majd meg.
```

Comment:
jelenleg a fordíthatóságot nem ellenőrzi

Reply:
Köszi, javítottam.